### PR TITLE
Event card

### LINKED ISSUE #2: Event home screen card
I'd like to have "cards" for each of our upcoming events appear on the homescreen. It's important the part of the card with the poster is as square as possible (its hard to accommodate a lot of different poster sizes). 

Screenshot 2021-05-11 at 5 53 17 PM

[https://carleton.ca/#content](https://carleton.ca/#content)

Screenshot 2021-05-11 at 6 00 59 PM

A design attempt I made:

Screenshot 2021-05-11 at 5 55 32 PM

### PR DESCRIPTION
**Pull request by EliasJRH**

### CODE REVIEW

File: layouts/partials/event_card.html [hunk 1]
```diff
@@ -1,3 +1,29 @@
-<div class="event_card">
-  <h2>{{ .Title }}</h2>
+<link rel="stylesheet" href="../../assets/scss/components/_event_card.scss" />
+<link rel="preconnect" href="https://fonts.gstatic.com" />
+<link
+  href="https://fonts.googleapis.com/css2?family=Nunito:wght@600&display=swap"
+  rel="stylesheet"
+/>
```
Comment (MathyouMB) [inline layouts/partials/event_card.html]: Please move these to head.html. Right now these lines are getting appended to the dom for every event card. We only need to link the font once.

File: layouts/partials/event_card.html [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+
+<div class="outer_card">
+  <div class="date_card">
+    <h3 style="text-align: center;">{{ .Params.date.Format "Jan" | upper }}</h3>
+    <h2 style="text-align: center;">{{ .Params.date.Format "2" }}</h2>
+    <h3 style="text-align: center;">{{ .Params.date.Format "2006" }}</h3>
```
Comment (MathyouMB) [inline layouts/partials/event_card.html:5]: [Question]

What is the Jan 2 2006 date?

**Review by MathyouMB — APPROVED** [review]
LGTM! I would just address the few comments I Ieft, and move any repetitive inline styling into css

File: layouts/partials/event_card.html (hunk 2 above)
Comment (EliasJRH) [reply layouts/partials/event_card.html:5]: According to the [Hugo dateFormat documentation](https://gohugo.io/functions/dateformat/) this syntax is how you're able to grab pieces of a date from a normal date string. Jan 2 2006 isn't any specific date and I'm not sure if the same functionality can be achieved using something other than this.

File: layouts/partials/event_card.html (hunk 1 above)
Comment (EliasJRH) [reply layouts/partials/event_card.html]: Right, got it.

**Pull request merged**